src/main/java/frc/robot/subsystems/shooter/commands/bits/ShootAtMaximalVelocity.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems.shooter.commands.bits;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class ShootAtMaximalVelocity extends CommandBase {
```

Comment:
Why do you need this command?

Reply:
1. To check whether the solenoid stays at the correct angle with maximal velocity.
2. To check the maximal distance that the shooter can reach.

Reply:
You don't really need this function, if I would tell the shooter to shoot at 2,000,000 rpm I don't think he would shoot at this rate...

Reply:
> To check whether the solenoid stays at the correct angle with maximal velocity.

WDYM?
> To check the maximal distance that the shooter can reach.

You can just use the `Shoot` command you wrote with a high, constant velocity setpoint.